### ISSUE TITLE
LuckyFlow::Element.fill not clear old value

### ISSUE THREAD

**Issue by skojin** [luckyframework/lucky_flow#43]
If I load in test 'edit' form with some 'Foo' value, and call fill with 'Bar', form element will have 'FooBar' value, expected just 'Bar'

**Comment by paulcsmith** [comment]
Could you provide some sample code? I’m not totally sure I understand

**Comment by skojin** [comment]
I have this flow, and I take screenshot for debug, I see this
```
    flow.visit CommandWatches::New, as: user
    flow.fill_form CommandWatchForm,
      name: "TestNewCommand",
      command: "date",
      notification_text: "Test $1"
    flow.click "@save-button"
    flow.el("table", text: "TestNewCommand").should be_on_page

    # edit
    flow.click "table a"
    flow.fill_form CommandWatchForm,
      name: "TestUpdatedCommand"
    flow.click "@save-button"
    flow.open_screenshot
    flow.el("table", text: "TestUpdatedCommand").should be_on_page
```
![1552929335](https://user-images.githubusercontent.com/8044/54549297-d18e0100-49ba-11e9-8001-c39052c53114.png)

**Comment by paulcsmith** [comment]
Oh I think I get it. So instead of replacing the text it is appending the text?

**Comment by paulcsmith** [comment]
Thanks for the screenshot and debug code! That was very helpful

**Comment by paulcsmith** [comment]
Fixed by #46. Thanks for reporting @skojin and thanks @edwardloveall for the fix
